src/app/features/collection/components/progress-bar-skeleton/progress-bar-skeleton.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="tw-flex tw-flex-col tw-items-center tw-w-full tw-p-1 tw-bg-zinc-50 tw-text-xs tw-text-black">
```

Comment:
it looks like we are using few rules for defining item background (tw-bg-zinc-50 and styles in scss file). If we can reduce rules ammount that wil be great. If no - let's keep it as it is